src/processRequest.mjs

```diff
@@ -275,7 +275,7 @@ export const processRequest = (
 
```

Comment:
While this line is never reached in node v8.x, it _is_ used in v10.x and v11.x. I'm not quite sure how to "fix" the code coverage here...